html/infrastructure/safe-passing-of-structured-data/transfer-errors.window.js

```diff
@@ -0,0 +1,42 @@
+function assert_transfer_error(transferList, exception = "DataCloneError") {
+  assert_throws(exception, () => self.postMessage({ get whatever() { throw exception } }, "*", transferList));
```

Comment:
nit: Is this trying too hard to do what could be two separate functions? Throwing the string "DataCloneError" as the default behavior seems odd.

Reply:
Will clean this up too.